external/fv3fit/fv3fit/emulation/transforms/zhao_carr.py

```diff
@@ -46,6 +50,61 @@
 QV_PRECPD = "specific_humidity_after_precpd"
 
 
+def limit_negative_cloud(
+    *, cloud: tf.Tensor, humidity: tf.Tensor, temperature: tf.Tensor,
```

Comment:
What is the extra args inclusion, `*`, for?

Reply:
It suppresses the use of positional arguments.